source/tls_channel_handler.c

```diff
@@ -388,6 +388,7 @@ int aws_tls_connection_options_set_alpn_list(
 
 int aws_channel_setup_client_tls(
     struct aws_channel *channel,
```

Comment:
I don't think you actually need this param anymore... Channel can be pulled off of the slot.

Reply:
Made a new PR with updates